src/lang.cls.php

```diff
@@ -206,6 +206,7 @@ public static function title( $id ) {
 			self::O_IMG_OPTM_AUTO => __('Auto Request Cron', 'litespeed-cache'),
 			self::O_IMG_OPTM_ORI => __('Optimize Original Images', 'litespeed-cache'),
 			self::O_IMG_OPTM_RM_BKUP => __('Remove Original Backups', 'litespeed-cache'),
+			self::O_IMG_OPTM_REP_W_SCALED => __('Replace Original with Scaled', 'litespeed-cache'),
```

Comment:
@timotei-litespeed The option needs to move to Media as well. Img optm is only for QC optm